aea/helpers/dialogue/base.py

```diff
@@ -510,10 +565,10 @@ def reply(self, target_message: Message, performative, **kwargs) -> Message:
 
```

Comment:
I think the `target_message` should be optional argument and by default (i.e. if not provided ink wargs) equal to the last message. That will simplify usage, as in most cases you reply to last message in dialogue

Reply:
Good idea.

Done